packages/block-library/package.json

```diff
@@ -20,8 +20,7 @@
 	"module": "build-module/index.js",
 	"react-native": "src/index",
 	"dependencies": {
-		"@babel/runtime": "^7.0.0-beta.52",
-		"@wordpress/api-fetch": "file:../api-fetch",
+		"@babel/runtime-corejs2": "7.0.0-beta.56",
```

Comment:
There are local changes to master after `npm install` with these changes. I think we missed an update to `package-lock.json`.

Reply:
I was wondering if there is a way to catch this kind of issue on Travis and fail the build. It isn’t first time when it happens.